packages/ozone/src/api/moderator/updateUser.ts

```diff
@@ -0,0 +1,42 @@
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+import { Server } from '../../lexicon'
+import AppContext from '../../context'
+import { getUserRole } from '../util'
+
+export default function (server: Server, ctx: AppContext) {
+  server.tools.ozone.moderator.updateUser({
+    auth: ctx.authVerifier.moderator,
```

Comment:
I think this is right, basing on the others:
```suggestion
    auth: ctx.authVerifier.modOrAdminToken,
```